
Do a hard reset to avoid terminal problems when the screen is refreshed.

--- include/config.h.ori	Tue Dec 10 03:05:55 1996
+++ include/config.h	Thu Sep  3 05:39:22 1998
@@ -495,7 +495,7 @@
 #undef EPIC_DEBUG		/* force coredump on panic */
 #define EXEC_COMMAND		/* allow /exec comamnd */
 #undef HACKED_DCC_WARNING	/* warn if handshake != sender */
-#undef HARD_UNFLASH		/* do a hard reset instead of soft on refresh */
+#define HARD_UNFLASH		/* do a hard reset instead of soft on refresh */
 #undef HOP			/* stuff i like that wintrhawk doesnt */
 #undef MURPLE			/* stuff i put in for murple to test */
 #undef NO_BOTS			/* no bots allowed */
